### PR TITLE
[IMP] public_budget: diario destino en vista tree transferencias

### PR DESCRIPTION
**Pull request by pablohmontenegro**
Task: 31482

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-public-administration/pull/493).

File: public_budget/views/account_payment_views.xml [hunk 1]
```diff
@@ -57,6 +57,18 @@
         </field>
     </record>
 
+    <record model="ir.ui.view" id="view_account_payment_tree">
+        <field name="name">account.payment.tree</field>
+        <field name="model">account.payment</field>
+        <field name="priority">120</field>
+        <field name="inherit_id" ref="account_payment_group.view_account_payment_from_group_tree"/>
+        <field name="arch" type="xml">
```
Comment (zaoral) [inline public_budget/views/account_payment_views.xml]: account.view_account_payment_tree

**Review by zaoral — CHANGES_REQUESTED** [review]

**Comment by zaoral** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-public-administration#509

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-public-administration#509

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-public-administration#509

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-public-administration#509

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-public-administration#509

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-public-administration#509

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-public-administration#509

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-public-administration#509

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-public-administration#509

**Pull request closed without merging**